src/App.js

```diff
@@ -1,61 +1,188 @@
+
 import React from 'react';
+import AddTodo from './components/AddTodo/AddTodo';
+import ShowTodos from './components/ShowTodo/ShowTodo';
+// import { runInThisContext } from 'vm';
+
+class App extends React.Component {
+  state = {
+    todosOriginal: [],
+    todosToShow: [],
+    activeTab: 'all',
+  }
+
+  addTodo = (newTodo) => {
+    const { activeTab } = this.state;
+
+    this.setState((prevState) => {
+      if (activeTab !== 'complete') {
```

Comment:
тут можно сделать 
```suggestion
      if (activeTab === 'complete') {
```
а ниже будет просто `elxe`
если условия два, то отлично будет